azure-pipelines.yaml

```diff
@@ -42,6 +44,16 @@ jobs:
   - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
     displayName: 'Component Detection'
 
+  - script: dotnet pack /p:PackageVersion=$(Major).$(Minor).$(Patch) -o '$(Build.ArtifactStagingDirectory)' --include-symbols src/Microsoft.Azure.WebJobs.Extensions.Kafka/Microsoft.Azure.WebJobs.Extensions.Kafka.csproj
+    displayName: 'Pack nuget pacakge'
+  
+  - task: PublishBuildArtifacts@1
+    inputs:
+      pathToPublish: '$(Build.ArtifactStagingDirectory)'
```

Comment:
Is this a shared directory somewhere? Or an Azure storage blob?

Reply:
This is reside on the Azure DevOps.  This task upload the artifacts to Azure DevOps (maybe storage or somewhere, not stated by them.) We can download it and use it from Release pipeline.  Generally speaking, Build and Release is different lifecycle, so Azure DevOps recommend to separate Build and Release pipeline. The best practice is , on the Build pipleline, we create an artifact (in this case nuget package) and upload by PublishBuildArtifact task, then Release pipeline refer this artifact and execute release (in this case publish to the nuget). We can't publish this week, so I just prepare for the next step.

Reply:
Ok, but we can download this manually after the CI is built? And then ask the product team to publish to NuGet manually for us?

Reply:
Yes. You can. Go to the CI summary page, you can see/download it.  It might depend on the talk with the product team next week, however, I recommend to automate it. Manual process is annoying for both you and product team. If they implement CI to their environment it might be OK. however, if it last a couple of months, automation might be better. e.g. After inviting him as a member of the AzureDevOps project, We can automate the pipeline and set the approver to him or he can push the button of the release.

Reply:
Yes, Automated is the eventual goal. For now, as long as we can do it manually, it's ok. Will revise this once we agree on where and how and who will auto publish this. Thanks.